superset-frontend/src/dashboard/util/getDashboardFilterKey.ts

```diff
@@ -16,11 +16,20 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-export function getDashboardFilterKey({ chartId, column }) {
+
+export interface GetDashboardFilterKeyProps {
```

Comment:
no need to export this since it's only being used in this file

Reply:
sure, updated